dev/tasks/verify-rc-alma/setup-AlmaLinux-Rocky-Linux.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh -l
```

Comment:
We should not have duplicated file in this repository.
We should use `dev/release/setup-...`.

Reply:
ok